### DISCUSSION TITLE
The function  createVirginQueens() should also work with just a mated queen (Pop) as the input

### DISCUSSION THREAD

**Discussion by janaobsteter** [HighlanderLab/SIMplyBee#281] [Q&A] [answered]

**Comment by gregorgorjanc** [comment]
I think I know where your coming from - to simplify the creation of a base population? Could you show an example here please?

**Comment by janaobsteter** [comment]
Yes, exactly. For example, in the simulation, I first create x virgin queens for the founders --> then use q for queens and (x - q) for drone producing queens. Then I mate the queens and in the next step I want to "graft/multiply" the queens to get another generation of virgin queens, but that is currently not possible without having the queen in a colony. I think this feature would also be helpful in the simulation of grafting from a single inseminated queen.

**Comment by janaobsteter** [comment]
```
founderGenomes <- quickHaplo(nInd = 5, nChr = 1, segSites = 100)
SP <- SimParamBee$new(founderGenomes)
basePop <- createVirginQueens(founderGenomes)
drones <- createDrones(x = basePop[1], nInd = 50)
queens <- crossVirginQueen(basePop[2:5], drones = drones, nFathers = 5)
createVirginQueens(queens)
```
which returns an error: "Error in createVirginQueens(queens) : 
  Argument x must be a Map-Pop or Colony or Colonies class object!"

**Comment by gregorgorjanc** [comment]
I think we had this before, but for some reason we went back to "you can only create virgin queens and workers from a colony". I am open to going back;) It`s and edge case, like with `createDrones()`. Maybe we make it clear in the docs that this is a hack?

**Comment by janaobsteter** [comment]
I will change it back then and make it clear in the documentation, that this should only be used in special cases ("multiplying" the founders, case studies of inheritance ...)

**Comment by janaobsteter** [comment]
So, we are not allowing this now (again). We need to discuss what we want here.

**Comment by janaobsteter** [comment]
I moved this to the discussion - @LStrachan , @JerBub

**Comment by gregorgorjanc** [comment]
@janaobsteter what is the status of this discussion?

**Comment by janaobsteter** [comment, marked as answer]
This function takes either a Map-Pop object or a Colony/MultiColony object.
